docs/Using_OpenWPM.md

```diff
@@ -34,80 +34,42 @@ for _ in range(num_browsers):
-  ```
-
-2. Define the behaviour of our new command in `*_commands.py` in `openwpm/commands/`,
-   e.g. `browser_commands.py`.
-   Feel free to add a new module within `openwpm/commands/` for your own custom commands  
-    In our case this looks like this:
-  ```python
-    from selenium.webdriver.common.action_chains import ActionChains
-
-    def jiggle_mouse(webdriver, number_jiggles):
-        for i in xrange(0, number_jiggles):
-            x = random.randrange(0, 500)
-            y = random.randrange(0, 500)
-            action = ActionChains(webdriver)
-            action.move_by_offset(x, y)
-            action.perform()
-  ```
-
-3. Make our function be called when the command_sequence reaches our Command, by adding it to the
-    `execute_command` function in `openwpm/commands/command_executer.py`
-      In our case this looks like this:
-  ```python
-        elif type(command) is JiggleCommand:
-        browser_commands.jiggle_mouse(
-            webdriver=webdriver,
-            number_jiggles=self.num_jiggles)
-  ```
-
-4. Lastly we change ```openwpm/CommandSequence.py``` by adding a `jiggle_mouse` method to the `CommandSequence`
-  so we can add our command to the commands list  
-  In our case this looks like this:
-  ```python
-    def jiggle_mouse(self, num_jiggles, timeout=60):
-    """ jiggles mouse <num_jiggles> times """
-    self.total_timeout += timeout
-    if not self.contains_get_or_browse:
-        raise CommandExecutionError("No get or browse request preceding "
-                                    "the jiggle_mouse command", self)
-    command = JiggleCommand(num_jiggles)
-    self.commands_with_timeout.append((command, timeout))
-  ```
-   A timeout is given and set by default to 60 seconds. This is added to the overall sequence timeout. Finally, we check that the `CommandSequence` instance contains a `get` or a `browse` command prior to this command being added by checking `self.contains_get_or_browse`. This is necessary as it wouldn't make sense to have selenium jiggle the mouse before loading a page.
-
-
-
-Notice that any arguments to the command are added both to the command sequence top-level method, and are then stored in the `Command` object to be serialized and sent across the process boundary between the task manager and browser manager.
-
-Finally, the command sequence given to the Task Manager to visit a site, sleep for 10 seconds, jiggle the mouse 10 times, and takes a screenshot would look like this:
-
+If you'd like to create a custom command, you must create a new file with the details
+of your command. Suppose we want to make a command to take a screenshot of the page we are viewing.
+In `custom_command.py` we would define a class, `SaveScreenshotCommand` that derives from `BaseCommand`
+```python
+class SaveScreenshotCommand(BaseCommand)
+```
+Next, in our newly derived class, we would define the `___init___`, `execute` and `__repr__` functions.
```

Comment:
Do these have a set interface? Do they need to accept a specific set of arguments? What's a `__repr__` function? 

These are the kinds of questions I'd expect someone new to have -- can we update this to point folks to the abstract base class (for a list of expected arguments) and also to provide guidance on the repr function, even if just a link out to some python docs.

Reply:
We have the `BaseCommand` as an interface that defines the `execute` function. Defining `__repr__` is just a python best practice (I suppose we could like to a document saying that). And an `__init__` is just required in this specific case as we want to store some data in the command. I'm unsure how to document that.

Reply:
I would add something like `see the Base class (link) for required arguments` and link `__repr__` to python docs. Try to make it as easy as possible for the reader :)